subsys/bluetooth/controller/ll_sw/lll_conn_iso.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2021 Demant
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct node_tx_iso {
+	union {
+		void        *next;
+		memq_link_t *link;
+	};
+
+	uint64_t payload_number; /* 39-bit cisPayloadNumber */
+	uint8_t  pdu[];
+};
+
+struct lll_conn_iso_strm_rxtx {
+	uint8_t phy;            /* PHY */
+	uint8_t burst_number;   /* Burst number (BN) */
+	uint8_t flush_timeout;  /* Flush timeout (FT) */
+	uint8_t max_octets;     /* Maximum PDU size */
+};
+
+struct lll_conn_iso_strm {
+	/* Link to ACL connection (for encryption, channel map, crc init) */
+	struct lll_conn *conn;
+
+	/* Connection parameters */
+	uint8_t  access_addr[4];    /* Access address */
+	uint32_t offset;            /* Offset of CIS from start of CIG in us */
+	uint32_t sub_interval;      /* Interval between subevents in us */
+	uint32_t subevent_length;   /* Length of subevent in us */
+	uint8_t  num_subevents;     /* Number of subevents */
+	struct lll_conn_iso_strm_rxtx rx; /* RX parameters */
+	struct lll_conn_iso_strm_rxtx tx; /* TX parameters */
+
+	/* Event and payload counters */
+	uint64_t event_count;       /* 39-bit cisEventCount */
+	uint64_t rx_payload_number; /* 39-bit cisPayloadNumber */
```

Comment:
Would it make sense to define these as 

```
	uint64_t event_count : 39;       /* 39-bit cisEventCount */
	uint64_t rx_payload_number : 39;  /* 39-bit cisPayloadNumber */
```
So that we cannot assign more than 39 bits to them (assuming that the compiler will catch value-out-of-bound errors).